registry/bin/boot

```diff
@@ -35,8 +37,7 @@ function etcd_set_default {
   ERROR="$(etcdctl --no-sync -C "$ETCD" mk "$ETCD_PATH/$1" "$2" 2>&1 >/dev/null)"
 
   if [[ $? -ne 0 ]] && echo "$ERROR" | grep -iqve "key already exists"; then
-    echo "etcd_set_default: an etcd error occurred ($ERROR)"
-    echo "aborting..."
+    echo "registry: etcd_set_default($1/$2) aborting, an etcd error occurred ($ERROR)"
```

Comment:
This line could be misleading.  Using the path separator / forward slash right there would yield a message that looks like this if, for instance, etcd failed setting _key_ `foo` to _value_ `bar`:

```
registry: etcd_set_default(foo/bar) aborting, an etcd error occurred...
```

That could be very misleading because `foo/bar` looks like a key, when really it's a key and value.  Could I suggest revising the format of that message?